public/components/trace_analytics/components/common/search_bar.tsx

```diff
@@ -36,22 +36,21 @@ export const renderDatePicker = (
   );
 };
 
-export interface SearchBarProps extends FiltersProps {
+export interface SearchBarProps {
+  filters: FilterType[];
```

Comment:
I saw these fields are added as non optional, but there are other references use SearchBarProps, have we double checked on other places as well?

Reply:
<img width="367" alt="Screenshot 2024-08-19 at 3 28 36 PM" src="https://github.com/user-attachments/assets/d84ed1f8-1f6c-4495-934a-dd8cca9aa86c">

Seems to be only used in those two files as an extension and for setting time. All seems to be working.